product-service/src/main/java/com/extremecoder/productservice/dto/ErrorResponseDto.java

```diff
@@ -0,0 +1,17 @@
+package com.extremecoder.productservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ErrorResponseDto {
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private String field;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Please use Validation api for each of such cases

Reply:
My plan was to prevent the null field from including in the JSON object. Can you please suggest how validation API can help me in this case?

Reply:
@kamrulbari14 Oh. i understand. So basically what you want here is not to include or ignore any json fields which is null. In this scenario java validation api won't help. Thanks for asking.

Reply:
Exactly @shshetudev bhai. Thanks for the concern. Actually, I felt confused, and then I had done some study if the Validation API can help me in this case. But I couldn't find any solution and that's why I asked for your suggestion. 😛

Reply:
Thank you bhai!